webpack.config.js

```diff
@@ -28,20 +29,28 @@ module.exports = {
     chunks: false
   },
   plugins: [
+    new webpack.optimize.UglifyJsPlugin(),
     new webpack.HotModuleReplacementPlugin(),
     new webpack.NamedModulesPlugin()
   ],
   module: {
     rules: [
+      // {
```

Comment:
Do you need this?

Reply:
maybe, not too sure, so would prefer to leave it because I will forget how to write it

Reply:
I have faith that you will figure it out. Please remove it.